test/functional/cmdLineTests/criu/src/org/openj9/criu/OptionsFileTest.java

```diff
@@ -293,6 +293,14 @@ static void jitOptionsTest(String[] args) {
 		System.out.println("Pre-checkpoint");
 		CRIUTestUtils.checkPointJVM(criuSupport, imagePath, true);
 		System.out.println("Post-checkpoint");
+
+		// Sleep to ensure that the JVM produces output needed for test success
```

Comment:
Is there a Java API that can query JIT server status such as "Connected to a server"? if so, it would be nice to have a deterministic way to know if `UseJITServer` is specified and a connection was successful.
Otherwise, waiting for `2s` seems a way to receive the expected console output.

Reply:
No doesn't look like there's any such java API.